src/drivers/stm32/drv_io_timer.c

```diff
@@ -423,12 +452,19 @@ int io_timer_init_timer(unsigned timer)
 			rBDTR(timer) = ATIM_BDTR_MOE;
 		}
 
-		/* If the timer clock source provided as clock_freq is the STM32_APBx_TIMx_CLKIN
-		 * then configure the timer to free-run at 1MHz.
-		 * Otherwize, other frequencies are attainable by adjusting .clock_freq accordingly.
```

Comment:
@kd0aij - Please preserve this comment 
```/* If the timer clock source provided as clock_freq is the STM32_APBx_TIMx_CLKIN
  * then configure the timer to free-run at 1MHz.
  * Otherwize, other frequencies are attainable by adjusting .clock_freq accordingly.
```
 - we need to tell dev's that are modifying the code to use the  .clock_freq  to get an different rPSC(timer) result, when hacking the range. If we do not we end up with ```#ifdefs```